package.json

```diff
@@ -47,6 +47,7 @@
     ]
   },
   "resolutions": {
+    "@types/react": "17.0.47",
```

Comment:
Does this mean that customers will have to resolve to this version in their apps as well?

Reply:
The resolution only ensures that all of our packages and deps resolve to `@react/types@17.0.47`. Has no impact on users

Reply:
I think I already asked this before, but why this specific version?

Reply:
Mostly to align with [this PR](https://github.com/aws-amplify/amplify-ui/pull/2413)

Reply:
The type files will be generated from 17. If we reference some types to build the library from React 17, customers still on 16 will run into missing types error. That's my concern. Actually we should pin it to 16.14 as part of the next breaking change

Reply:
Discussed offline